Return-Path: <git-commits-head-owner@vger.kernel.org>
X-Original-To: max@dual
Delivered-To: max@dual
Received: from dual (localhost [127.0.0.1])
	by dual (Postfix) with ESMTP id 987DC2443E
	for <max@dual>; Tue, 29 Apr 2008 09:33:25 +0200 (CEST)
X-Original-To: janitor@sternwelten.at
Delivered-To: janitor@sternwelten.at
Received: from baikonur.stro.at [213.239.196.228]
	by dual with POP3 (fetchmail-6.3.8)
	for <max@dual> (single-drop); Tue, 29 Apr 2008 09:33:25 +0200 (CEST)
Received: from vger.kernel.org (vger.kernel.org [209.132.176.167])
	by baikonur.stro.at (Postfix) with ESMTP id 6E0285C04F
	for <janitor@sternwelten.at>; Tue, 29 Apr 2008 03:59:36 +0200 (CEST)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1756259AbYD2CB2 (ORCPT <rfc822;janitor@sternwelten.at>);
	Mon, 28 Apr 2008 22:01:28 -0400
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S1753672AbYD2CB2
	(ORCPT <rfc822;git-commits-head-outgoing>);
	Mon, 28 Apr 2008 22:01:28 -0400
Received: from hera.kernel.org ([140.211.167.34]:33325 "EHLO hera.kernel.org"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1756259AbYD2CB1 (ORCPT
	<rfc822;git-commits-head@vger.kernel.org>);
	Mon, 28 Apr 2008 22:01:27 -0400
Received: from hera.kernel.org (IDENT:U2FsdGVkX18k968SizcbE8WWe89TWsqsrQUDklFpb90@localhost [127.0.0.1])
	by hera.kernel.org (8.13.8/8.13.8) with ESMTP id m3T21QiF014336
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NO)
	for <git-commits-head@vger.kernel.org>; Tue, 29 Apr 2008 02:01:26 GMT
Received: (from dwmw2@localhost)
	by hera.kernel.org (8.13.8/8.13.1/Submit) id m3T21QAh014319
	for git-commits-head@vger.kernel.org; Tue, 29 Apr 2008 02:01:26 GMT
Date:	Tue, 29 Apr 2008 02:01:26 GMT
Message-Id: <200804290201.m3T21QAh014319@hera.kernel.org>
From: Linux Kernel Mailing List <linux-kernel@vger.kernel.org>
To: git-commits-head@vger.kernel.org
Subject: piix: add Asus Eee 701 controller to short cable list
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
X-Git-Commit: 1fa5a40f27e832b7c31e93d6dc2952a41ed277f7
X-Git-Parent: 2064c946eaca641d28281264d01d0c8b977f1f20
Sender: git-commits-head-owner@vger.kernel.org
Precedence: bulk
List-ID: <git-commits-head.vger.kernel.org>
X-Mailing-List:	git-commits-head@vger.kernel.org
X-Virus-Scanned: by Amavis (ClamAV) at stro.at
Content-Length: 1394

Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=1fa5a40f27e832b7c31e93d6dc2952a41ed277f7
Commit:     1fa5a40f27e832b7c31e93d6dc2952a41ed277f7
Parent:     2064c946eaca641d28281264d01d0c8b977f1f20
Author:     Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
AuthorDate: Mon Apr 28 23:44:43 2008 +0200
Committer:  Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
CommitDate: Mon Apr 28 23:44:43 2008 +0200

    piix: add Asus Eee 701 controller to short cable list
    
    Based on ata_piix patch by Dan McGee.
    
    Signed-off-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
---
 drivers/ide/pci/piix.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/ide/pci/piix.c b/drivers/ide/pci/piix.c
index 21c5dd2..f04738d 100644
--- a/drivers/ide/pci/piix.c
+++ b/drivers/ide/pci/piix.c
@@ -250,6 +250,7 @@ static const struct ich_laptop ich_laptop[] = {
 	{ 0x27DF, 0x1043, 0x1267 },	/* ICH7 on Asus W5F */
 	{ 0x27DF, 0x103C, 0x30A1 },	/* ICH7 on HP Compaq nc2400 */
 	{ 0x24CA, 0x1025, 0x0061 },	/* ICH4 on Acer Aspire 2023WLMi */
+	{ 0x2653, 0x1043, 0x82D8 },	/* ICH6M on Asus Eee 701 */
 	/* end marker */
 	{ 0, }
 };
--
To unsubscribe from this list: send the line "unsubscribe git-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
